README.md

```diff
@@ -2,6 +2,8 @@
 
 A project based learning activity for people who are getting started with Git and GitHub.
 
+### Instructions for playing the game
+
 You can play the game at: https://githubschool.github.io/github-games/
 
```

Comment:
you can use the up and down arrow key to navigate and also left and right arrows to change the position